install.sh

```diff
@@ -60,24 +71,29 @@ if ! sudo apt-get install gpg wget; then
     die_cmd "Failed to check if needed packages are installed."
 fi
 
-echo
-msg "Multiple releases of makedeb are available for installation."
-msg "Currently, you can install one of 'makedeb', 'makedeb-beta', or"
-msg "'makedeb-alpha'."
-msg "If you're not sure which one to choose, pick 'makedeb'."
+if [[ "${MAKEDEB_PKG:+x}" == '' ]]; then
+    echo
+    msg "Multiple releases of makedeb are available for installation."
+    msg_valid_makedeb_pkgs
+    msg "If you're not sure which one to choose, pick 'makedeb'."
 
-while true; do
-    read -p "$(question "Which release would you like? ")" response
+    while true; do
+        read -p "$(question "Which release would you like? ")" response
 
-    if [[ "${response}" != 'makedeb' && "${response}" != 'makedeb-beta' && "${response}" != 'makedeb-alpha' ]]; then
-        error "Invalid response: ${response}"
-        continue
-    fi
+        if ! is_valid_makedeb_pkg "$response"; then
+            error "Invalid response: ${response}"
+            continue
+        fi
 
-    break
-done
+        break
+    done
 
-MAKEDEB_PKG="${response}"
+    MAKEDEB_PKG="${response}"
+elif ! is_valid_makedeb_pkg "$MAKEDEB_PKG"; then
+    error "Invalid \$MAKEDEB_PKG: $MAKEDEB_PKG"
+    msg_valid_makedeb_pkgs
```

Comment:
@hwittenborn I pushed a new change reusing listing the valid MAKEDEB_PKG values also in case of an error, WDYT?

Reply:
I'm definitely down with that, let's keep it!